packages/core/src/renderer/util.ts

```diff
@@ -0,0 +1,158 @@
+// Refactored version of `makeIdsUnique` in https://github.com/iconfu/svg-inject/blob/master/src/svg-inject.js
```

Comment:
A bit pedantic but I'd replace the `master` reference here with a specific commit:

```suggestion
// Refactored version of `makeIdsUnique` in https://github.com/iconfu/svg-inject/blob/064ac002930deaf96eefb95eaf953c5ef5287992/src/svg-inject.js
```